lab1.js

```diff
@@ -106,19 +124,27 @@ assert(nextAnimal, 'assign something to nextAnimal');
 */
 
 // number of times the new caretaker fed the lion. one array entry per day
-var mealsPerDay = [5, 4, 3, 6, 2, 4, 3, 4, 5, 1];
+var mealsPerDay = [ 5, 4, 3, 6, 2, 4, 3, 4, 5, 1 ];
 var tooHungryDay;
 
-/*
- TODO: 20 points
- Cycle through the days in mealsPerDay. At each day, print out the average
- number of meals/day the lion got since the new caretaker started.
- tooHungryDay should receive the number of days before the lion started
- pondering protein supplements (the first day the average dips below 4
- meals)
-*/
+var mealsTotal = 0;
+var avgMealsDay = Math.round(mealsTotal / (i + 1) * 100) / 100;
```

Comment:
don't really need this whole expression here, since you define it in the for loop. You can just declare
var avgMealsDay
and leave it undefined for now.